images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -133,7 +133,10 @@ function Get-VcpkgVersion {
     $result = Get-CommandResult "vcpkg version"
     $result.Output -match "version (?<version>\d+\.\d+\.\d+)" | Out-Null
     $vcpkgVersion = $Matches.version
-    return "Vcpkg $vcpkgVersion"
+    Push-Location -Path "/usr/local/share/vcpkg"
```

Comment:
nitpick: it is not necessary to cd to folder before using git.
You can just invoke `git -C "/usr/local/share/vcpkg" rev-parse master`. See details in https://stackoverflow.com/a/35899275

Reply:
thank you for the nitpick, changed to suggested variant